packages/ruby_rubocop.rb

```diff
@@ -0,0 +1,49 @@
+# Adapted from Arch Linux ruby-rubocop PKGBUILD at:
+# https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?h=ruby-rubocop
+
+require 'package'
+
+class Ruby_rubocop < Package
+  description 'A Ruby static code analyzer and formatter'
+  homepage 'https://rubocop.org'
+  version '1.22.0'
+  compatibility 'all'
+  source_url 'SKIP'
+
+  depends_on 'libyaml'
+
+  @xdg_config_home = ENV['XDG_CONFIG_HOME']
+  @xdg_config_home = "#{CREW_PREFIX}/.config" if @xdg_config_home.to_s.empty?
+
+  def self.install
+    ENV['CREW_FHS_NONCOMPLIANCE_ONLY_ADVISORY'] = '1'
+    warn_level = $VERBOSE
+    $VERBOSE = nil
+    load "#{CREW_LIB_PATH}lib/const.rb"
+    $VERBOSE = warn_level
+    FileUtils.mkdir_p "#{CREW_DEST_DIR}#{@xdg_config_home}/rubocop"
+  end
+
+  def self.postinstall
+    system 'gem install -N rubocop --conservative'
```

Comment:
Shouldn't this go in the `self.install` section so files can be tracked for removal?  Otherwise, removing the package leaves the gem installed.  Not ideal.

Reply:
I added self.remove for this.  I don't want to interfere with the gem installed database.